jenkins/tests/test_timeout.py

```diff
@@ -63,7 +65,7 @@ def test_stop_stuck_builds_with_stuck(self, mock_desc,
 
         stop_build.assert_has_calls([call(3), call(2)], any_order=True)
 
-        update_desc.assert_hass_calls(
```

Comment:
How did this pass before this?!

Reply:
The old version of mock just treated it like any other function call on a Mock and returned a new Mock (so it didn't actually do an assertion at all).  The new version is smart enough to realize "hmm, that looks like you were trying to call a function that actually exists..." and raise an error.